Doc/library/venv.rst

```diff
@@ -234,11 +234,12 @@ creation according to their needs, the :class:`EnvBuilder` class.
 There is also a module-level convenience function:
 
 .. function:: create(env_dir, system_site_packages=False, clear=False, \
-                     symlinks=False, with_pip=False)
+                     symlinks=False, with_pip=False, prompt=None)
 
     Create an :class:`EnvBuilder` with the given keyword arguments, and call its
     :meth:`~EnvBuilder.create` method with the *env_dir* argument.
 
+    .. versionadded:: 3.3
```

Comment:
Thanks for the PR. The `prompt` parameter was added in fd0f84bd8f1 and hence ``versionadded`` directive should be 3.6 . You can take reference from the commit where ``versionadded`` directive is added for `EnvBuilder`

```
.. versionadded:: 3.6
   Added the ``prompt`` parameter
```

Reply:
This actually marks the function itself as being added in 3.3. 
If updated the PR to also include an entry for the `prompt` parameter.

Reply:
Added blank lines.

Concerning the version: the venv PEP targets 3.3 and I cannot find any reference to *venv* in the trees for 3.1 and 3.2. That said GH shows the 3.1 tag on the commit you mentioned - but since there is no actually code there I'd say 3.3 is correct

Reply:
You're right. I went ahead with information from GitHub UI and `git tag --contains` also showed me 3.1 was the version locally. I checked out the tags and I couldn't find any references to EnvBuilder. Sorry for the confusion.

```
$ git tag --contains 7ded1f0 | grep 3.1 # but not present in the tags
3.1
v3.3.1
v3.3.1rc1
```